### PR TITLE
Make android great again

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

The new version of gomobile no longer passes an `ios` tag when building, so I added that in.

@zanderz Can you please review my changes to the secret_store_external?

@akalin-keybase I added the serviceName stuff, could you tal, and make sure it looks ok?
I'm not doing any testing on the secret store stuff yet. I'll make a separate ticket to do that.

### CODE REVIEW

File: shared/login/forms/intro.render.native.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react'
+import {View, Text} from 'react-native'
+
+export default class Render extends Component {
+  render () {
+    return (
+      <View>
+        <Text>TODO 123</Text>
```
Comment (MarcoPolo) [inline shared/login/forms/intro.render.native.js]: @chrisnojima When I change this, the packager doesn't pick it up unless I restart the packager.

File: react-native/android/app/build.gradle [hunk 2]
```diff
@@ -9,46 +12,65 @@ apply plugin: 'com.android.application'
...
+def enableSeparateBuildPerCPUArchitecture = true
+
+/**
+ * Run Proguard to shrink the Java bytecode in release builds.
+ */
+def enableProguardInReleaseBuilds = false
+
 android {
     compileSdkVersion 23
     buildToolsVersion "23.0.1"
```
Comment (jacobhaven) [inline react-native/android/app/build.gradle]: `23.0.2` is the current version installed with the SDK, we might want to update that here.

**Comment by zanderz** [comment]
LGTM, sorry again for the breakage :thumbsup:

File: shared/login/forms/intro.render.native.js (hunk 1 above)
Comment (chrisnojima) [inline shared/login/forms/intro.render.native.js]: yeah i made a ticket to look into this

**Comment by chrisnojima** [comment]
the js stuff seems fine to me :+1:

**Comment by akalin-keybase** [comment]
I bless this serviceName code. Go forth and commit, my child. :+1: :angel:

**Pull request merged**